docs/gettingStarted.md

```diff
@@ -10,48 +10,11 @@ This guide will provide you with all information you need to get started.
 
 Use the following accounts to test the application
 
-| Account Guid                         | Description                                              |
-| ------------------------------------ | -------------------------------------------------------- |
-| 77da92db-0791-491e-8c58-1a969e67d2fa | An account with a linked CSO account and an efiling role |
+| Account Guid | Description |
+| --- | --- |
+| 77da92db-0791-491e-8c58-1a969e67d2fa | An account with a linked CSO account and efiling role |
 | 77da92db-0791-491e-8c58-1a969e67d2fb | An account with a linked CSO account and no efiling role |
-| 88da92db-0791-491e-8c58-1a969e67d2fb | An account without CSO account but a fake bceid profile  |
-
-## Quick start
-
-Generate a secure url by calling the generateUrl endpoint
-
-```bash
-
-curl --location --request POST 'http://localhost:8080/document/generateUrl' \
---header 'Content-Type: application/json' \
---data-raw '{
-  "documentMetadata": {
-    "type": "string",
-    "subType": "string",
-    "documentAccess": {
-      "url": "string",
-      "verb": "GET",
-      "headers": {
-        "additionalProp1": "string",
-        "additionalProp2": "string",
-        "additionalProp3": "string"
-      }
-    }
-  },
-  "navigation": {
-    "success": {
-      "url": "string"
-    },
-    "error": {
-      "url": "string"
-    },
-    "cancel": {
-      "url": "string"
-    }
-  }
-}'
-
-```
+| 88da92db-0791-491e-8c58-1a969e67d2fb | Bob Ross, a user without CSO account but a bceid profile |
```

Comment:
```suggestion
| 88da92db-0791-491e-8c58-1a969e67d2fb | Bob Ross, a user without a CSO account but with a BCeID profile |
```

Reply:
Hey @sdevalapurkar-bcgov the other guys are not Bob Ross, they are `temp` and `temp` I need some recommendations to give them the name they deserve, @adodge-bcg @akroon3r @TayGov @KFloodBCGov @karthikmsqa

Reply:
Ross had one son, Robert Stephen "Steve" Ross,[33] with his first wife, Vivian Ridge. Steve, also a talented painter, occasionally appeared on The Joy of Painting and became a Ross-certified instructor.[12] Steve appeared on camera in the last episode of Season 1, in which he read a series of general "how-to" questions sent in by viewers during the season, and Bob answered them one at a time, technique by technique, until he had completed an entire painting.

Ross and Ridge's marriage ended in divorce in 1977. Ross and his second wife, Jane, had no children together. In 1992, Jane died of cancer. In 1995, two months before his death, Ross married for a third time, to Lynda Brown.[34]

Ross was very secretive about his life and had a great liking for privacy, with only a tight circle of friends. Some of only a few interviews with his close-knit circle of friends and his family can be found in the 2011 PBS documentary Bob Ross: The Happy Painter.[11]

His company, Bob Ross Inc., is protective of his intellectual property and his privacy to this day.

Reply:
As per this article I went with bob ross wives but scrambled their last names ;-), thanks @akroon3r for your deep analyse on bob